src/routes/JobCreateRoute.js

```diff
@@ -9,7 +9,16 @@ class JobCreateRoute extends React.Component {
   static manifest = Object.freeze({
     jobs: {
       type: 'okapi',
-      path: 'erm/jobs/packageImport',
+      path: (_q, _p, _r, _l, props) => {
```

Comment:
This could be cleaned up with the suggested Route change.

Reply:
Cleaned up, but might need more tweaks...?